### ISSUE TITLE
question abou the preprocessing

### ISSUE THREAD

**Issue by whubaichuan** [Cly1st/ECG-Arrhythmia-Classification-using-Artificial-Neural-Network#3]
Hi, thanks for this wonderful code. But still I have three questions:

(1) why do you resample from 360 to 128?

(2) It seems that you do not do any normalization for the data. Now the range of data is not from 0 to 1. It is from roughly 400 to 1000. Do you think moralization is better?

(3) do you think the metric `accuracy` is reasonable or not? Because I think that in test set `normal` is 44238 samples,  `Fusion` is 388 samples, `Supreventricular Ectopic ` is 1836 samples, `Ventricular Ectopic` is 3221 samples, `Unknown` is 7 samples, where `normal` will contribute almost 90% to `accuracy`. That means that even `Fusion`, `Supreventricular Ectopic `,  `Ventricular Ectopic` and `Unknown` are totally detect wrongly, the final `accuracy` could be 90% or more. (Plus, the training set is also unbalanced.)
<img width="560" alt="截屏2023-03-03 13 28 37" src="https://user-images.githubusercontent.com/28725662/222720377-09a0f485-7d55-44a9-b8af-3bd8985df8e6.png">

**Comment by Cly1st** [comment]
Hello @whubaichuan, thanks for being interested in this project. Let me answer your questions here:

1. The reason behind down sampling is before I started this project I do have planned to embed this model into an edge device (resource constraints).
2. I agreed with you, that normalization might produce a much better result. Since it has been proven by many researchers.
3. Again, you have a good point here. This dataset is imbalanced, you can apply the resampling technique on this dataset to make this model more generalized; Also we can check on another Performance Metric as well. (e.g. Precision, Recall, or ROC Curve).
